.github/workflows/backport-pull-request.yml

```diff
@@ -0,0 +1,39 @@
+---
+name: backport-pull-request
+on:
+  pull_request_target:
+    types:
+      - closed
+      - labeled
+
+jobs:
+  backport:
+    name: Backport
+    runs-on: ubuntu-latest
+    # Only react to merged PRs for security reasons.
+    # See https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target.
+    env:
+      backport-label: apply-
+    if: >
+      github.event.pull_request.merged
+      && (
+        github.event.action == 'closed'
+        || (
+          github.event.action == 'labeled'
+          && contains(github.event.label.name, env.backport-label)
```

Comment:
```suggestion
          && github.event.label.name == 'auto-backport'
```

Reply:
Adding the additional condition
```yaml
      && contains(join(github.event.issue.labels.*.name, ', '), env.backport_label_prefix)`
```
seem good enough for now, sage enough to try for me.